ADDsynth.adoc

```diff
@@ -38,9 +38,12 @@ image::imgs/lfo.png[]
 - *Start:* LFO Start Phase - If this knob is at the lowest value, the LFO Start Phase will be random.
 - *Str.:* LFO Stretch
 - *Delay:* LFO Delay
+- *LP:* Cutoff frequency of the lowpass filter applied to the lfo output. (i.e. how much the output is smoothed) 
+- *F.In:* Time it takes to fade in the lfo amplitude up to depth after the delay time 
+- *F.Out:* Time it takes to fade the amplitude out after key release.
 - *A.R.:* LFO Amplitude Randomness
 - *F.R.:* LFO Frequency Randomness
-- *Sync:* ???
+- *Sync:* if this option enabled the lfo oscillates in phase for all notes
```

Comment:
Just checking, this is what it does, right? (It sounds obvious, but I swear I had one or more oscillate at a different rate, perhaps due to randomness)

Reply:
phase = fmod((float)t.time() * phaseInc, 1.0f);
so you are right. it (obviously) also depends on the frequency.